### ISSUE TITLE
Issue uploading file greater than 50GB

### ISSUE THREAD

**Issue by taylorwmj** [ncw/rclone#613]
rclone v1.32-18-gf7562dcβ
Ubuntu Server 16.04.1
ACD
`rclone sync -v --stats=10s --low-level-retries=1 --acd-upload-wait-time=2m0s --max-size=70G --log-file=/home/... --transfers=2 /tmp/.PlexMedia/" "MediaServerBackup:/PMS/
`
[Log](https://github.com/ncw/rclone/files/415732/rclone_log_20160812.txt)

Now before we continue, _I realize_ the stated max file size has been 50GB. However, that was established over 9 months ago, and I'm wondering if there has been a recent change in this. 

I only ask because I stumbled upon [this](https://forums.developer.amazon.com/articles/21942/what-is-the-max-upload-for-a-file-to-cd.html) post from an Amazon rep that stats there is no limit on max upload size via the API. This post is nearly a year old, but I found it intriguing that they would say this. 

My upload of a roughly 54GB file fails, but only during the time-out period. Question I have is this: is rclone limiting my file size and then deleting it at the end of the failed upload, when it perhaps uploaded correctly, or is this still being limited by Amazon?

**Comment by ncw** [comment]
No, rclone doesn't limit your file upload size.  It is possible Amazon have raised the limit - let me know if you succeed with files > 50GB.

**Comment by taylorwmj** [comment]
Wasn't able to succeed with a filer larger than 50GB. Don't know if Amazon is actually failing tho, because it looks similar to other large files in that it starts through the `--acd-upload-wait-time` processing and then just ends with no issues being stated regarding size save for the rclone warning. 

I've attached the [log](https://github.com/ncw/rclone/files/420251/rclone_log_20160815.txt). The entire log is for just one file; apparently it kept trying to upload it, even after failing. 

Any idea? or is this Amazon's limitation it's now hitting. The log doesn't seem to provide the most detail here.

**Comment by ncw** [comment]
Checking the log out, each time where it managed to upload 100% of the file, it gave a "500 Internal Server Error" which I think is Amazon's way of saying it isn't supported :-(

**Comment by taylorwmj** [comment]
I have a feeling you're right. Do you know the best way to get an answer regarding this? I've tried to reach out to the API support via email, but they never responded. Just looking to get a firm answer on where they currently are sitting at.

**Comment by ncw** [comment]
Amazon have been strangely coy about publishing file size limits.  50GB is the limit most talked about though.  Try asking customer support rather than API support.
